api/app/Extensions/Revisionable/ChangeloggableTrait.php

```diff
@@ -0,0 +1,282 @@
+<?php
```

Comment:
@xiphiaz I wasn't sure how tightly coupled you wanted the package to be with Spira, or if it should be kept more universal Laravelesque. I'll keep it all in the package and remove the extended methods from Spira.